GitUI/Translation/French.xlf

```diff
@@ -9202,7 +9202,7 @@ when all items are shown:</source>
       <trans-unit id="lblMenuEntries.Text">
         <source>Select items to be shown in the cascaded context menu.
 (Unchecked items will be shown top level for direct access.)</source>
-        <target>Sélectionneé les items à afficher dans
+        <target>Sélectionnée les items à afficher dans
```

Comment:
Sélectionner, not Sélectionnée. 
In French: il faut utiliser l'infinitif, pas le participe passé.